docs/docs/guides/artifact.mdx

```diff
@@ -1,27 +1,25 @@
 ---
-title: Creating an Artifact
+title: How to generate configuration artifacts from templates
 ---
 
 import VideoPlayer from '../../src/components/VideoPlayer';
 
-# Creating an artifact in Infrahub
+# How to generate configuration artifacts from templates
```

Comment:
I think we should specific Jinja2 templates to avoid confusion with Infrahub templates